apps/src/ailab/Ailab.js

```diff
@@ -54,7 +54,7 @@ Ailab.prototype.init = function(config) {
   config.pinWorkspaceToBottom = true;
 
   const onMount = () => {
-    // NOTE: Other apps call studioApp.init(), except WebLab. Fish is imitating WebLab
+    // NOTE: Other apps call studioApp.init(), except WebLab. Ailab is imitating WebLab.
```

Comment:
So, is it now true that all other apps, except Fish AND Weblab AND AiLab, call studioApp.init()?